pkg/gardenlet/operation/botanist/secrets.go

```diff
@@ -305,11 +305,25 @@ func (b *Botanist) generateObservabilityIngressPassword(ctx context.Context) err
 		return err
 	}
 
+	// TODO(oliver-goetz): Remove `url` when Gardener v1.110 is released.
+	annotations := map[string]string{
+		"url":         "https://" + b.ComputePlutonoHost(),
+		"plutono-url": "https://" + b.ComputePlutonoHost(),
+	}
+
+	if b.IsShootMonitoringEnabled() {
```

Comment:
I think we can omit this check as the `generateObservabilityIngressPassword` func is called only when `b.WantsObservabilityComponents()` returns true: https://github.com/gardener/gardener/blob/bdbcd5df9556c94d1db971a7a7267535b425ff34/pkg/gardenlet/operation/botanist/secrets.go#L61-L65